dask/dataframe/tests/test_dataframe.py

```diff
@@ -5770,3 +5770,20 @@ def test_to_backend():
         # Moving to a "missing" backend should raise an error
         with pytest.raises(ValueError, match="No backend dispatch registered"):
             df.to_backend("missing")
+
+
+@pytest.mark.parametrize("func", ["first", "last", "max", "sum"])
+def test_transform_getitem_works(func):
+    df = pd.DataFrame({"ints": [1, 2, 3], "grouper": [0, 1, 0]})
+
+    ddf = dd.from_pandas(df, npartitions=2)
+    meta = df.groupby("grouper").transform(func)
+    df["new"] = df.groupby("grouper").transform(func)["ints"]
+    ddf["new"] = ddf.groupby("grouper").transform(func, meta=meta)["ints"]
+
+    # for first/last, the values will be different from pandas, because Dask
+    # shuffled the records in each group, so we fix the data to match
+    if func in ("first", "last"):
+        df["new"] = ddf["new"] = 1
```

Comment:
If we need to do this for these specific functions, should we include them at all in the test?

Reply:
Good point, they don't add much there. I removed them.